generated/nidcpower/nidcpower/lcr_measurement.py

```diff
@@ -0,0 +1,229 @@
+        ("rp"                  , nidcpower._visatype.ViReal64 ),  # noqa: E202,E203
+        ("d"                   , nidcpower._visatype.ViReal64 ),  # noqa: E202,E203
+        ("q"                   , nidcpower._visatype.ViReal64 ),  # noqa: E202,E203
+        ("measurement_mode"    , nidcpower._visatype.ViUInt16 ),  # noqa: E202,E203
+        ("dc_in_compliance"    , nidcpower._visatype.ViBoolean),  # noqa: E202,E203
+        ("ac_in_compliance"    , nidcpower._visatype.ViBoolean),  # noqa: E202,E203
+        ("unbalanced"          , nidcpower._visatype.ViBoolean),  # noqa: E202,E203
+        ("reserved1"           , nidcpower._visatype.ViReal64 ),  # noqa: E202,E203
+        ("reserved2"           , nidcpower._visatype.ViReal64 ),  # noqa: E202,E203
+        ("reserved3"           , nidcpower._visatype.ViReal64 ),  # noqa: E202,E203
+        ("reserved4"           , nidcpower._visatype.ViReal64 ),  # noqa: E202,E203
+        ("reserved5"           , nidcpower._visatype.ViReal64 ),  # noqa: E202,E203
+        ("reserved6"           , nidcpower._visatype.ViReal64 ),  # noqa: E202,E203
+        ("reserved7"           , nidcpower._visatype.ViReal64 ),  # noqa: E202,E203
+    ]
+
+
+class LCRMeasurement(object):
+    """Specifies an LCR measurement.
+
+    Fields:
+        channel (str): The channel name associated with this LCRMeasurement.
+
+        vdc (float): Specifies the measured DC voltage, in volts.
+
+        idc (float): Specifies the measured DC current, in amps.
+
+        stimulus_frequency (float): Specifies the frequency of the LCR test signal, in Hz.
+
+        ac_voltage (complex): Specifies the measured AC voltage, in volts RMS.
+
+        ac_current (complex): Specifies the measured AC current, in amps RMS.
+
+        z (complex): Specifies the complex impedance.
+
+        series_lcr (LCR): Specifies the inductance, in henrys, the capacitance, in farads, and
+            the resistance, in ohms, as measured using a series circuit model.
+
+        parallel_lcr (LCR): Specifies the inductance, in henrys, the capacitance, in farads, and
+            the resistance, in ohms, as measured using a parallel circuit model.
+
+        d (float): The dissipation factor of the circuit.
+
+        measurement_mode (enums.InstrumentMode): Specifies the measurement mode.
+            **Defined Values**:
+
+            +-----------------------+-----------------------------------------------------+
+            | InstrumentMode.SMU_PS | The channel(s) are operating as a power supply/SMU. |
+            +-----------------------+-----------------------------------------------------+
+            | InstrumentMode.LCR    | The channel(s) are operating as an LCR meter.       |
+            +-----------------------+-----------------------------------------------------+
+
+        in_compliances (InCompliances): Indicates whether the output was in DC compliance and/or
```

Comment:
consider having:

dc_in_compliance
ac_in_compliance

as bools

Reply:
Zen of Python:

> Flat is better than nested.

Reply:
> Flat is better than nested.

This seems to contradict with our discussions during our previous LCR Python API review meeting (based on what I recall), as the initial design for this custom struct had all the members laid out flatly, but then we decided to group related fields into namedtuples (e.g. `series_lcr` and `parallel_lcr`). 

Anyway, I have split this into two bools in my latest commit.